projects/addon-mobile/directives/dropdown-mobile/dropdown-mobile.options.ts

```diff
@@ -0,0 +1,10 @@
+import {tuiCreateOptions} from '@taiga-ui/cdk/utils/di';
+
+export interface TuiDropdownMobileOptions {
+    topShift: number;
+}
+
+export const [TUI_DROPDOWN_MOBILE_OPTIONS, tuiDropdownMobileOptionsProvider] =
```

Comment:
Let's add a CSS variable instead of this, seems cleaner.

Reply:
I've introduced `--t-dropdown-mobile-margin-top` css variable

https://github.com/taiga-family/taiga-ui/pull/10980/commits/5081f80fa7f3e7975b4436568d8d05c0362d2c20

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/10980/commits/00d570ec911940b1f05ef00332e6ea0b8141237c